src/components/macroview/centerPanel/SequencingArea.tsx

```diff
@@ -158,33 +156,29 @@ export default function SequencingArea({ onOpenSettingsModal }: Props) {
         </Stack>
       </VStack>
       <HStack
-        justifyContent="right"
+        justifyContent="center"
         w="full"
         alignItems="center"
         px={[2, 4, 6]}
       >
+        <Text fontWeight="semibold" fontSize={['sm', 'md']}>
+          Sequence
+        </Text>
         <Button
           variant="brandRecord"
           leftIcon={recording ? <StopIcon /> : <RecordIcon />}
           size={['xs', 'sm', 'md']}
+          fontSize={['xs', '13.5', 'lg']}
           isActive={recording}
           onClick={recording ? stopRecording : startRecording}
         >
           {recording ? 'Stop' : 'Record'}
         </Button>
         <Button
-          variant="brandWarning"
-          leftIcon={<DeleteIcon />}
-          size={['xs', 'sm', 'md']}
-          onClick={onOpen}
-          isDisabled={sequence.length === 0}
-        >
-          Clear All
-        </Button>
-        <Button
-          variant="brandAccent"
+          variant="brandRecord"
           leftIcon={<TimeIcon />}
           size={['xs', 'sm', 'md']}
+          fontSize={['xs', '13.5', 'lg']}
```

Comment:
You should be very careful specifying an exact number here. I'm pretty sure that the named values will be `em/rem` which will scale with the users global font size. An exact number won't, so it would be better to use a named value or an em/rem value

Reply:
Theres a couple other instances of it in this file. Maybe you could even put this into a const in the function and reuse it, as these bits of text are intended to be the same size

Reply:
Tried tweaking here, set it to an equivalent rem value: [88dad27](https://github.com/WootingKb/wooting-macros/pull/194/commits/88dad27e27fa4fda0c7ed11c080bd91ecf95b347)